concert/tests/unit/test_parameter.py

```diff
@@ -36,6 +36,24 @@ def _set_foo(self, value):
     param = Parameter(fget=_get_foo, fset=_set_foo)
 
 
+class FooDeviceTagetValue(BaseDevice):
+    foo = Quantity(q.mm)
+
+    def __init__(self, value):
+        super(BaseDevice, self).__init__()
+        self._value = value
+
+    def _get_foo(self):
+        delta = np.random.randint(low=1, high=100) * q.mm
```

Comment:
Why do you need randomness?

Reply:
I had the impression, that it is more realistic in this way, but a fixed value should also work.